README.md

```diff
@@ -1,10 +1,20 @@
-# ENV SSM
-Gets params from your environment first, then from the ssm parameter store.
+# env-ssm
 
-## Options
+Load environment from SSM Parameter Store.
+
+Variables load (and are overwritten if duplicates are found) in the following order:
 
-| Option | Description |
-|---| --- |
-| region | The AWS SSM region where the parameters are stored. |
-| prefix | The prefix to attach to the beginning of the parameter name. Can also be the key of an environment variable who's value will be the prefix. |
+1. Load `SSM`
+2. Load `.env` file
+3. Load `.tfvars` file
+4. Load `process.env`
+
+## Options
```

Comment:
It's nice to see the options, but I'd also like to see a usage example, including how to require or import it.

Reply:
Added 👍🏻